pkg/services/sqlstore/org_users.go

```diff
@@ -189,7 +199,8 @@ func (ss *SQLStore) SearchOrgUsers(ctx context.Context, query *models.SearchOrgU
 
 	// get total count
 	orgUser := models.OrgUser{}
-	countSess := x.Table("org_user")
+	countSess := x.Table("org_user").
+		Join("INNER", x.Dialect().Quote("user"), fmt.Sprintf("org_user.user_id=%s.id", x.Dialect().Quote("user")))
```

Comment:
Which databases can't handle "org_user.user_id=user.id"?  Is that Access?

Reply:
not sure, but the tests would not get through If I did not get the dialect correct